src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -209,6 +300,16 @@ private static bool IsRawBodyBytesRequested(Capabilities capabilities)
             return capabilities.GetCapabilityState(LanguageWorkerConstants.RawHttpBodyBytes) != null;
         }
 
+        private static bool IsTypeDataCollectionSupported(Capabilities capabilities)
+        {
+            string typeDataCollectionSupported = capabilities.GetCapabilityState(LanguageWorkerConstants.TypedDataCollectionSupported);
```

Comment:
simplify: 
```
return capabilities.GetCapabilityState(LanguageWorkerConstants.TypedDataCollectionSupported) != null;
```

Reply:
Is it better for string testing use string.IsNullOrEmpty?